migrations/0015_create_messages_table.up.sql

```diff
@@ -0,0 +1,16 @@
+create table messages (
+    id int unsigned not null auto_increment primary key,
+    sender_id int not null,
+    recipient_id int null,
+    recipient_channel varchar(255) null,
+    content varchar(2048) not null,
+    unread boolean not null default 0,
```

Comment:
unread with default false? default is read?

Reply:
its only marked as unread if they are offline on bancho rn, maybe unread is an unfitting name